src/msw/window.cpp

```diff
@@ -5860,6 +5915,36 @@ bool wxWindowMSW::HandlePressAndTap(const wxPoint& pt, WXDWORD flags)
 
     return HandleWindowEvent(event);
 }
+
+bool wxWindowMSW::HandleTouch(WXWPARAM wParam, WXLPARAM lParam)
+{
+    const unsigned count = LOWORD(wParam);
+    TOUCHINPUT info[count];
```

Comment:
This is not standard C++, we need to use `wxVector` here to allocate an array of dynamic size.